Gemfile.jruby-2.3.lock.release

```diff
@@ -186,96 +186,96 @@ GEM
       rspec (~> 3.0)
       rspec-wait
       stud (>= 0.0.20)
-    logstash-filter-aggregate (2.7.0)
+    logstash-filter-aggregate (2.7.2)
       logstash-core-plugin-api (>= 1.60, <= 2.99)
-    logstash-filter-anonymize (3.0.5)
+    logstash-filter-anonymize (3.0.6)
       logstash-core-plugin-api (>= 1.60, <= 2.99)
       murmurhash3
-    logstash-filter-cidr (3.1.1-java)
+    logstash-filter-cidr (3.1.2-java)
       logstash-core-plugin-api (>= 1.60, <= 2.99)
-    logstash-filter-clone (3.0.4)
+    logstash-filter-clone (3.0.5)
       logstash-core-plugin-api (>= 1.60, <= 2.99)
-    logstash-filter-csv (3.0.6)
+    logstash-filter-csv (3.0.7)
       logstash-core-plugin-api (>= 1.60, <= 2.99)
-    logstash-filter-date (3.1.8)
+    logstash-filter-date (3.1.9)
       logstash-core-plugin-api (>= 1.60, <= 2.99)
-    logstash-filter-de_dot (1.0.2)
+    logstash-filter-de_dot (1.0.3)
       logstash-core-plugin-api (>= 1.60, <= 2.99)
-    logstash-filter-dissect (1.1.1)
+    logstash-filter-dissect (1.1.2)
       jar-dependencies
       logstash-core-plugin-api (>= 2.1.1, <= 2.99)
-    logstash-filter-dns (3.0.6)
+    logstash-filter-dns (3.0.7)
       logstash-core-plugin-api (>= 1.60, <= 2.99)
       lru_redux (~> 1.1.0)
-    logstash-filter-drop (3.0.4)
+    logstash-filter-drop (3.0.5)
       logstash-core-plugin-api (>= 1.60, <= 2.99)
-    logstash-filter-elasticsearch (3.2.0)
+    logstash-filter-elasticsearch (3.2.1)
       elasticsearch (>= 5.0.3, < 6.0.0)
       logstash-core-plugin-api (>= 1.60, <= 2.99)
-    logstash-filter-fingerprint (3.1.1)
+    logstash-filter-fingerprint (3.1.2)
       logstash-core-plugin-api (>= 1.60, <= 2.99)
       murmurhash3
-    logstash-filter-geoip (5.0.1-java)
+    logstash-filter-geoip (5.0.2-java)
       logstash-core-plugin-api (>= 1.60, <= 2.99)
-    logstash-filter-grok (3.4.3)
+    logstash-filter-grok (3.4.5)
```

Comment:
Does this need to be 4.0.0 now as 3.4.5 has been yanked

Reply:
indeed this should be removed